src/components/auth/torus/PasswordLess.js

```diff
@@ -0,0 +1,115 @@
+import React, { useCallback } from 'react'
+import { getDesignRelativeHeight } from '../../../lib/utils/sizes'
+import { isSmallDevice } from '../../../lib/utils/mobileSizeDetect'
+import Section from '../../common/layout/Section'
+import CustomButton from '../../common/buttons/CustomButton'
+import { theme as mainTheme } from '../../theme/styles'
+import normalizeText from '../../../lib/utils/normalizeText'
+import mobileBtnIcon from '../../../assets/Auth/btn_mobile.svg'
+import { withStyles } from '../../../lib/styles'
+import { fireEvent, SIGNIN_METHOD_SELECTED, SIGNUP_METHOD_SELECTED } from '../../../lib/analytics/analytics'
+import { LoginButton } from '../login/LoginButton'
+import useTorus from './hooks/useTorus'
+
+const ShowPasswordless = ({ isSignup = true, isOpen, styles, onSelect, handleLoginMethod }) => {
+  const [, sdkInitialized] = useTorus()
+
+  // const _email = useCallback(() => {
+  //   handleLoginMethod('auth0-pwdless-email')
+  // })
+
+  const _mobile = useCallback(() => {
+    handleLoginMethod('auth0-pwdless-sms')
+  })
```

Comment:
```suggestion
  const _mobile = useCallback(() => {
    handleLoginMethod('auth0-pwdless-sms')
  }, [handleLoginMethod])
```

Also, please use the LoginStrategy enum items instead of text strings
e.g. `LoginStrategy.PaswordlessSMS` etc